app/views/pets/show.html.erb

```diff
@@ -10,7 +10,7 @@
       <p>Approximate Age: <%= @pet.approx_age %><br/>
       Sex: <%= @pet.sex %><br/>
       Status: <%= @pet.status %><br/>
-      <%= "On hold for #{@pet.approved_for}" unless @pet.approved_for.nil? %>
+      <%= "Sorry this pet is on hold" unless @pet.application_approved.nil? %>
```

Comment:
User Story 22 says we should have the name of the applicant here:
"And I see text on the page that says who this pet is on hold for (Ex: "On hold for John Smith", given John Smith is the name on the application that was just accepted)"

Maybe we should store the name of the applicant in the application_approved column instead of the application id so we can call @pet.application_approved